docs/content/en/docs/tasks/workload/ingress.md

```diff
@@ -29,10 +27,10 @@ We currently recommend using Emissary-ingress Kubernetes Ingress Controller by A
     kubectl apply -f "https://anywhere.eks.amazonaws.com/manifests/hello-eks-a.yaml"
     ```
 
-2. Set up kube-vip service type: Load Balancer in your cluster by following the instructions [here]({{< ref "/docs/tasks/workload/loadbalance#setting-up-kube-vip-for-service-type-load-balancer" >}}).
-Alternatively, you can set up MetalLB Load Balancer by following the instructions [here]({{< ref "/docs/tasks/workload/loadbalance#alternatives" >}})
+2. Set up Kube-Vip service type: LoadBalancer in your cluster by following the instructions [here]({{< ref "/docs/tasks/workload/loadbalance#setting-up-kube-vip-for-service-type-load-balancer" >}}).
+Alternatively, you can set up MetalLB by following the instructions [here]({{< ref "/docs/tasks/workload/loadbalance#alternatives" >}}).
 
-3. Install Ambassador CRDs and ClusterRoles and RoleBindings
+3. Install Ambassador CRDs and ClusterRoles and RoleBindings.
```

Comment:
```suggestion
3. Install Ambassador CRDs, ClusterRoles and RoleBindings.
```

Reply:
Fixed in next revision to include a comma before "and" as well